### PR TITLE
Docs: Update docs to add network benchmarks for tcp and gpcnet

### PR DESCRIPTION
**Pull request by yukirora**
**Description**
Update docs to add network benchmarks for tcp and gpcnet.

### CODE REVIEW

File: docs/user-tutorial/benchmarks/micro-benchmarks.md [hunk 1]
```diff
@@ -130,6 +130,52 @@ Support the following operations currently: allreduce, allgather, broadcast, red
...
+
+Test the TCP connectivity between current node and nodes in the hostfile,
+performed by [tcping](https://github.com/zhengxiaowai/tcping)
+
+#### Metrics
+
+| Metrics                                      | Unit     | Description                                                                          |
+| -------------------------------------------- | -------- | ------------------------------------------------------------------------------------ |
+| tcp-connectivity/Successed_${hostname/ip}    | count    | successed times of tcp connections between current node and other nodes              |
+| tcp-connectivity/Failed_${hostname/ip}       | count    | failed times of tcp connections between current node and other nodes                 |
```
Comment (abuccts) [inline docs/user-tutorial/benchmarks/micro-benchmarks.md:145]: why there's hostname/ip in metrics?

File: docs/user-tutorial/benchmarks/micro-benchmarks.md [hunk 2]
```diff
@@ -130,6 +130,52 @@ Support the following operations currently: allreduce, allgather, broadcast, red
...
+| {benchmark_name}/${test_title}_GetBcast(4096B)_${stat}              | bandwidth (MB/s/rank) | statistical values(Min, Max, Avg, 99%, 99.9%) obtained by all nodes use congestion 'Get Bcast(4096B)' for congestion testing                                               |
+| {benchmark_name}/${test_title}_PutIncast(4096B)_${stat}             | bandwidth (MB/s/rank) | statistical values(Min, Max, Avg, 99%, 99.9%) obtained by all nodes use congestion 'Put Incast (4096 B)' for congestion testing                                            |
+| {benchmark_name}/${test_title}_Two-sidedIncast(4096B)_${stat}       | bandwidth (MB/s/rank) | statistical values(Min, Max, Avg, 99%, 99.9%) obtained by all nodes use congestion 'Two-sided Incast (4096 B)' for congestion testing                                      |
+| {benchmark_name}/${test_title}_Alltoall(4096B)_${stat}              | bandwidth (MB/s/rank) | statistical values(Min, Max, Avg, 99%, 99.9%) obtained by all nodes use congestion 'Alltoall (4096 B)' for congestion testing                                              |
```
Comment (abuccts) [inline docs/user-tutorial/benchmarks/micro-benchmarks.md:176]: do we really need all output numbers for metrics?

File: docs/user-tutorial/benchmarks/micro-benchmarks.md (hunk 1 above)
Comment (cp5555) [reply docs/user-tutorial/benchmarks/micro-benchmarks.md:145]: Yes. I have the same question.

File: docs/user-tutorial/benchmarks/micro-benchmarks.md (hunk 2 above)
Comment (cp5555) [reply docs/user-tutorial/benchmarks/micro-benchmarks.md:176]: I think we can do it because it's not for people to understand.

**Review by cp5555 — APPROVED** [review]

**Review by abuccts — APPROVED** [review]

**Pull request merged**